test/e2e/tests/tokens/nft/mocks.js

```diff
@@ -411,51 +411,6 @@ function setupAutoDetectMocking(
         json: ensTokenResponse,
       };
     });
-
-  // eth_blockNumber
-  server
-    .forPost('/v3/00000000000000000000000000000000')
-    .withBodyIncluding('eth_blockNumber')
-    .thenCallback(() => {
-      return {
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: 1111111111111111,
-          result: '0x1',
-        },
-      };
-    });
-
-  // eth_getBlockByNumber
-  server
-    .forPost('/v3/00000000000000000000000000000000')
-    .withBodyIncluding('eth_getBlockByNumber')
-    .thenCallback(() => {
-      return {
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: 1111111111111111,
-          result: {},
-        },
-      };
-    });
-
-  // eth_call
-  server
-    .forPost('/v3/00000000000000000000000000000000')
-    .withBodyIncluding('eth_call')
-    .thenCallback(() => {
-      return {
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: 1111111111111111,
-          result: '0x1',
-        },
-      };
-    });
```

Comment:
These mocks were actually making the nft e2e fail. Unsure why.

Reply:
mm interesting. Those mocks are quite old.
It's possible that eth getBlockByNumber and eth_blocknumber are not needed anymore as RPC calls as we now rely almost entirely on API instead of relying in direct contracts calls /RPC methods.
About the failure,, given that the mock for eth_call is very generic, it could be that with these changes we are performing an eth_call (maybe to the balance checker?), which is then captured by this mock unintentionally, causing the spec to fail?